bin/pre-commit-hook.js

```diff
@@ -38,6 +38,19 @@ function linterFailure() {
 	process.exit( 1 );
 }
 
+function phpcsInstalled() {
+	try {
+		execSync( 'which -s phpcs phpcbf' );
+		return true;
+	} catch ( error ) {
+		console.log( chalk.red( 'PHP files will not be processed because PHPCS was not found.' ) );
```

Comment:
For follow up: a link to docs with setup instructions could be helpful here.

Reply:
https://github.com/Automattic/wp-calypso/issues/34552